### PR TITLE
LPS-164088 Extract versions to API

### PR DESCRIPTION
**Pull request by jkappler**
## Motivation for LPS-164088

[Link to story](https://issues.liferay.com/browse/LPS-164088)

We have extracted content dashboard item to API. So to clean the API more, we want to also extract the internal class Versions to it's own class

## Change summary:

* Extract internal class Versions in ContentDashboardItem to it's own class
* Change usages
* Standardize namings

## Notes

* Unit tests were changed accordingly

### CODE REVIEW

**Comment by jkappler** [comment]
ci:test:sf

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:test:tango

File: modules/apps/content-dashboard/content-dashboard-api/src/main/java/com/liferay/content/dashboard/item/ContentDashboardItemVersion.java
Comment (cgoncas) [inline modules/apps/content-dashboard/content-dashboard-api/src/main/java/com/liferay/content/dashboard/item/ContentDashboardItemVersion.java:28]: Do you think we should sort the parameters in alphabetical order?

**Review by cgoncas — APPROVED** [review]
LGTM, I only one doubt about if we should sort the parameters of the constructor in alphabetical order.

File: modules/apps/content-dashboard/content-dashboard-api/src/main/java/com/liferay/content/dashboard/item/ContentDashboardItemVersion.java
Comment (jkappler) [reply modules/apps/content-dashboard/content-dashboard-api/src/main/java/com/liferay/content/dashboard/item/ContentDashboardItemVersion.java:28]: I'll change this to avoid issues @cgoncas, thanks!

**Comment by jkappler** [comment]
ci:stop

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:forward

**Comment by jkappler** [comment]
@beltranrengifo, we'll send LPS-163792 after this has been merged, it's a nightmare with SF, even after ant all 😭

**Comment by beltranrengifo** [comment]
> @beltranrengifo, we'll send LPS-163792 after this has been merged, it's a nightmare with SF, even after ant all 😭

Good @jkappler no problemo!
I'll send a new pr, the old one got stucked, still indicates de sf running LOL

**Comment by jkappler** [comment]
ci:reopen

**Comment by jkappler** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:forward

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:forward

**Pull request closed without merging**
